packages/plugins/auto-doc/src/pages/template-overview/template-overview.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts" context="module">
+
+    export type Template = {
+        name: string,
+        description: string,
+        lastEdited: Date,       
+    }
+</script>
+
+<script lang="ts">
+    import {Table} from '@/components'
+
+    let allTemplates : Template[] = [
+        {
+            name: 'Template1',
+            description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Assumenda, asperiores accusamus quis sapiente sequi facilis?',
+            lastEdited: new Date("2024-10-15T08:38:00"),
+        },
+        {
+            name: 'Template256',
+            description: 'A simple description',
+            lastEdited: new Date("2024-10-10T13:24:00")
+        },
+        {
+            name: 'CYK_Template',
+            description: 'I have an invalid Date ',
+            lastEdited: new Date("foo"),
+        },
+        {
+            name: 'I surely have quite a long title name.',
+            description: "Lorem ipsum dolor sit amet consectetur.",
+            lastEdited: new Date("2024-10-20T10:55:32"),
+        },
+        {
+            name: 'AUOBHBT Template',
+            description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta placeat itaque minus praesentium sequi deleniti voluptatibus dicta!',
+            lastEdited: new Date(),
+        },
+    ]    
+</script>
+
+<main class="tamplate-overview">
```

Comment:
Typo in the class name : `tamplate-overview` ;)
I use a VSCode extension to highlight these : Code Spell Checker (it comes with multiple language support)

```suggestion
<main class="template-overview">
```